dashboard/src/components/authentication/login/resetPassword.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect } from 'react';
+import TextField from '@material-ui/core/TextField';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import CardHeader from '@material-ui/core/CardHeader';
+import Footer from '../../common/footer/footer';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Please do the above change here also.

Reply:
@yashodgayashan made the required changes